tests/RAGchain/benchmark/dataset/test_triviaqa.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+import pathlib
+
+import pytest
+from langchain.llms.openai import OpenAI
+
+from RAGchain.DB import PickleDB
+from RAGchain.benchmark.dataset import TriviaQAEvaluator
+from RAGchain.pipeline import BasicRunPipeline
+from RAGchain.retrieval import BM25Retrieval
+
+root_dir = pathlib.PurePath(os.path.dirname(os.path.realpath(__file__))).parent.parent.parent
+bm25_path = os.path.join(root_dir, 'resources', 'bm25', 'ko_strategy_qa_evaluator.pkl')
+pickle_path = os.path.join(root_dir, 'resources', 'pickle', 'ko_strategy_qa_evaluator.pkl')
```

Comment:
pkl file name must be different to each test evaluators.
Fix other test files if they use ko_strategy_qa_evaluator as well.